### DISCUSSION TITLE
Task 08 - Problem2

### DISCUSSION THREAD

**Discussion by noreengao0924** [chulminy/CIVE497-CIVE700#176] [Q&A]
Hi, can I use Relu for both hidden layers in this question? or do I need to use Relu for one of the layers, and other functions for the other layer?

**Comment by chulminy** [comment]
Replace Relu for all nodes in hidden layers. 

Chul Min

**Reply by m269zhou** [reply]
The question asked to implement the ReLU activation function in one of the hidden layers. Does that mean I should do this?
![Weixin Image_20240417144442](https://github.com/chulminy/CIVE497-CIVE700/assets/155658659/95a9912a-a978-4950-8313-9a712eb490f8)

**Reply by chulminy** [reply]
the activation functions are on the node, not arrow. You can do that but change both (Except for the softmax for the last layer). 

Chul Min

**Reply by m269zhou** [reply]
So, we should change the activation function in both two hidden layers not just one of two hidden layers.

**Reply by chulminy** [reply]
Either one is fine. But changing both are easier.

Chul Min
